roles/libvirt_manager/tasks/clean_layout.yml

```diff
@@ -56,7 +56,7 @@
       community.libvirt.virt:
         command: undefine
         flags:
-          - keep_nvram
+          - nvram
```

Comment:
@hjensas thoughts on that? AFAIK we can get rid of nvram all the time without any issue - correct? We're using overlays anyway, so the VM itself is undefined, UUID and other things change on each creation... ?

Reply:
yes, I think we should clean nvram. It will be re-created when defining new VMs.